ncm-metaconfig/src/main/metaconfig/irods/pan/schema.pan

```diff
@@ -0,0 +1,73 @@
+declaration template metaconfig/irods/schema;
+
+include 'pan/types';
+
+@{ type for configuring the irods_environment.json config file for clients @}
+type irods_environment_client_config = {
+    'irods_host' :  type_hostname
```

Comment:
Minor syntax suggestions:
* Remove space before ':'
* Replace ' by " (like most of the other lines)